src/js/electron/menu/searchFieldContextMenu.js

```diff
@@ -64,6 +64,7 @@ export default function searchFieldContextMenu(
       menu.separator(),
       searchMenuActions.fromTime.menuItem([field], {enabled: isTime}),
       searchMenuActions.toTime.menuItem([field], {enabled: isTime}),
+      searchMenuActions.jumpToTime.menuItem([field], {enabled: isTime}),
```

Comment:
If you want access to the log in your menu action listener,  you could add it after `field`: [field, log]. Then it will the last argument in the listener above.